Docs/Book/RDKit_Book.rst

```diff
@@ -251,6 +251,8 @@ The features which are parsed include:
   ``Q_e``, ``QH_p``, ``AH_P``, ``X_p``, ``XH_p``, ``M_p``, ``MH_p``, ``*``)
 - atomic properties ``atomprop``
 - coordinate/dative bonds ``C`` (these are translated into dative bonds)
+- hydrogen bonds ``H``
```

Comment:
don't see any related change for this feature. am i missing something?

Reply:
> don't see any related change for this feature. am i missing something?

H has been there for a while, it just wasn't in the docs

Reply:
thanks!